vendor/vendor.json

```diff
@@ -110,7 +110,7 @@
 			"revisionTime": "2017-09-28T14:21:59Z"
 		},
 		{
-			"checksumSHA1": "UUnaKjQAEIclOm5Aqe2VmrMiQJY=",
+			"checksumSHA1": "jK98PjsZS3gPOZfc+pqIQaz6r1A=",
```

Comment:
SHA1 is forbidden for checksums, as it is trivial to fake. Does govendor support anything better? SHA256? SHA512? SHA3? Blake3?

Reply:
mmm I guess I don't know why this changed locally for me, I see that the revision did not change so it's not like the dependency should actually have changed, but upon a fresh checkout in a clean GOPATH, this keeps changing, so I'm not sure what happened here

Reply:
IIRC the checksum calculation changed in a recent update of govendor  and the archive and snap versions computed different values for the same package. Is it possible that you're mixing different versions?

Reply:
I thought govendor development stopped a year ago? And recommendation to switch to go-modules when possible.

What recent update of govendor? Can you point me to those changes?

Reply:
We cannot switch to go modules yet because we still have to support building with vendorized dependencies on distros that don't have a new enough Go to support go modules IIRC

Reply:
I've been trying to track down what's changed but I can't figure out why my machine still keeps producing this SHA, I tried with govendor from focal deb pkg in a fresh / pristine GOPATH checkout of snapd master, and running `get-deps.sh` still modifies vendor.json